pkg/crypto/tinkcrypto/primitive/cl/signer/cl_signer_key_manager.go

```diff
@@ -18,84 +18,90 @@ import (
 	tinkpb "github.com/google/tink/go/proto/tink_go_proto"
 	clsubtle "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/cl/subtle"
 	clpb "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/proto/cl_go_proto"
+	"github.com/hyperledger/aries-framework-go/pkg/internal/ursautil"
 	"github.com/hyperledger/ursa-wrapper-go/pkg/libursa/ursa"
```

Comment:
usually our import packages are grouped in such order:
1. core Go deps
2. external deps
3. aries framework go deps

i suggest moving ursal-wrapper-go dep up with `golang/protobuf/proto` and `google/tink/go/keyset`